core/include/scipp/core/variable.h

```diff
@@ -53,7 +53,7 @@ using VariableConceptHandle = VariableConceptHandle_impl<
 ///
 /// This is the most generic representation for a multi-dimensional array of
 /// data. More operations are supportd by the partially-typed VariableConceptT.
-class VariableConcept {
+class SCIPP_CORE_EXPORT VariableConcept {
```

Comment:
Is this required? I think `VariableConcept` should only be an internal and is not required to be visible to other libraries.

Reply:
It doesn't link without this...